src/frontegg-mock/src/lib.rs

```diff
@@ -91,6 +96,20 @@ impl FronteggMockServer {
             ])
         });
 
+        // Provide default roles if None is provided
+        let roles = roles.unwrap_or_else(|| {
+            vec![
+                UserRole {
+                    id: "1".to_string(),
+                    name: "Organization Admin".to_string(),
+                },
+                UserRole {
+                    id: "2".to_string(),
+                    name: "Organization Member".to_string(),
```

Comment:
Can we default to the real values we use in production? I think the `id` for those would be `MaterializePlatformAdmin` and `MaterializePlatform`. I'm not 100% sure of this, as the field is called `key` rather than `id` in the frontegg python API. In the very few places we're looking at roles rather than permissions, these are what we're looking for. I think the `name` is correct, it's just the `id` that doesn't match.

In the cloud repo kind tests, we use the following args to match the production roles:
```yaml
          args:
            - "--listen-addr=0.0.0.0:6880"
            - "--issuer=http://127.0.0.1:32018"
            - "--encoding-key-file=/etc/jwk/private.pem"
            - "--decoding-key-file=/etc/jwk/public.pem"
            - '--users=[{"email": "infra+cloud-integration-tests-kind-cloud-0@materialize.io", "password": "asdf1234", "tenant_id": "77764ce8-c9f1-440b-ab08-5a6fbdc3ab96", "initial_api_tokens": [], "roles": ["MaterializePlatformAdmin", "MaterializePlatform"]}]'
```
If you change the default role `id`s, we'll need fewer changes downstream in the cloud repo.

Reply:
Ah yes good point! So the `id` is just an uuid. I am using that in the Terraform provider to store resources in the state file. Just introduced `key` as an additional field here as well!